doc/source/categorical.rst

```diff
@@ -515,6 +515,45 @@ To get a single value `Series` of type ``category`` pass in a list with a single
 
     df.loc[["h"],"cats"]
 
+String and datetime accessors
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 0.17.1
+
+The accessors  ``.dt`` and ``.str`` will work if the ``s.cat.categories`` are of an appropriate
+type:
+
+
+.. ipython:: python
+
+    str_s = pd.Series(list('aabb'))
+    str_cat = str_s.astype('category')
+    str_cat.str.contains("a")
+
+    date_s = pd.Series(date_range('1/1/2015', periods=5))
+    date_cat = date_s.astype('category')
+    date_cat.dt.day
+
+The returned ``Series`` (or ``DataFrame``) is of the same type as if you used the
+``.str.<method>`` / ``.dt.<method>`` on a ``Series`` of that type. In the above case, the
+returned values of ``str_cat.str.contains("a")`` and ``str_s.str.contains("a")`` or
+``date_cat.dt.day`` and ``date_s.dt.day`` will be equal:
```

Comment:
Would it be ok to make (half of ) this paragraph into note and insert a `(and not of type "category")` like this:

```
.. note::

    The returned ``Series`` (or ``DataFrame``) is of the same type as if you used the 
    ``.str.<method>`` / ``.dt.<method>`` on a ``Series`` of that type (and not of type ``category``). 

That means, that in the above case, the returned values of ``str_cat.str.contains("a")`` and ``str_s.str.contains("a")`` or ``date_cat.dt.day`` and ``date_s.dt.day`` will be equal:

...
```